include/atomics/receiverCadmium.hpp

```diff
@@ -28,85 +37,133 @@
 using namespace cadmium;
 using namespace std;
 
-//Port definition
-    struct Receiver_defs{
-        struct out : public out_port<Message_t> {
-        };
-        struct in : public in_port<Message_t> {
-        };
+/** 
+* Structure that holds input and output messages.
+*/
+struct receiver_defs{
+    struct out : public out_port<message_t> {
+    };
+    struct in : public in_port<message_t> {
     };
-   
-    template<typename TIME>
-    class Receiver{
-        using defs=Receiver_defs; // putting definitions in context
-        public:
-            //Parameters to be overwriten when instantiating the atomic model
-            TIME   preparationTime;
-            // default constructor
-            Receiver() noexcept{
-              preparationTime  = TIME("00:00:10");
-              state.ackNum    = 0;
-              state.sending     = false;
-            }
+};
+
+/** 
+* The Receiver class receives message and sends out acknowledge.
+*/   
+template<typename TIME>
+class Receiver{
+    using defs=receiver_defs; // putting definitions in context
+    public:
+        //Parameters to be overwriten when instantiating the atomic model
+        TIME   PREPARATION_TIME;
```

Comment:
Please add comments about the time constant

Reply:
Done